src/main/scala/net/snowflake/spark/snowflake/Parameters.scala

```diff
@@ -670,31 +670,6 @@ object Parameters {
       */
     def postActions: Array[String] = parameters(PARAM_POSTACTIONS).split(";")
 
```

Comment:
Some explain to reviewers:
These two functions are not used. And they are in Parameter. I don't think any customer should use them.